src/misc/support.cpp

```diff
@@ -80,6 +80,18 @@ bool ends_with(const std::string &suffix, const std::string &str) noexcept
 	return std::equal(sfx, sfx + n, txt, txt + n);
 }
 
+bool is_executable_filename(const std::string &filename) noexcept
+{
+	const size_t n = filename.length();
+        if (n < 4)
+		return false;
+	if (filename[n - 4] != '.')
+		return false;
+        std::string sfx = filename.substr(n - 3);
```

Comment:
Spaces sneaked in :) (line 86, 90, maybe others as well)

Reply:
argh - good catch! I need to figure out how to have vscode show me these prefix spaces.